### ISSUE TITLE
css_strategy

### ISSUE THREAD

**Issue by granthusbands** [markets/invisible_captcha#52]
The two css strategies that simply position things offscreen or use overflow to hide them are not accessible for blind users (and accessibility is a legal requirement in the US). A screen reader will read it out and invite the user to fill it in. The display:none one should be fine, though.

**Comment by JohnVillalovos** [comment]
Accessibility is a legal requirement for businesses employing 15 or more full-time employees. Or a business which is a public accommodation, like a hotel.

For everyone else it is not illegal.

That doesn't mean they shouldn't fix it to make it work for blind users of course.

https://www.businessnewsdaily.com/10900-ada-website-requirements.html

**Comment by markets** [comment]
Hi @granthusbands @JohnVillalovos for the ping and the info, really appreciated! As per the docs, the gem ships by default with a "special" label for the honeypot input, see the [`sentence_for_humans`](https://github.com/markets/invisible_captcha#plugin-options) option: 

> If you are a human, ignore this field

This can be customized even with [i18n](https://github.com/markets/invisible_captcha#i18n). So in case the input becomes visible (there is also an option to make the honeypot visibles `visual_honeypots`), the user reading that form will read the advise.
